src/main/java/org/takes/facets/hamcrest/HmRsHeader.java

```diff
@@ -43,78 +46,180 @@
  * @author Eugene Kondrashev (eugene.kondrashev@gmail.com)
  * @version $Id$
  * @since 0.23.3
- * @todo #260:30min Implement additional constructors.
- *  According to #260 there should be also available such constructors
- *  public HmRsHeader(final Matcher<? extends Map.Entry<String,String>> mtchr);
- *  public HmRsHeader(final String header,
- *  final Matcher<? extends Iterable<String>> mtchr);
- *  public HmRsHeader(final String header,
```

Comment:
@pinaf The puzzle in the code was a bit outdated. The final decision about the constructors signature is defined [here](https://github.com/yegor256/takes/issues/260#issuecomment-139857888).